From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Thomas A Caswell <tcaswell@gmail.com>
Date: Wed, 13 Mar 2024 14:05:35 -0400
Subject: [PATCH] 00424: gh-116745: remove all internal usage of @LIBPYTHON@

Follow on to https://github.com/python/cpython/pull/115780 .

Without this change numpy's build is broken.
---
 Misc/python-config.sh.in | 2 +-
 Misc/python.pc.in        | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Misc/python-config.sh.in b/Misc/python-config.sh.in
index eb02223ddc..c3c0b34fc1 100644
--- a/Misc/python-config.sh.in
+++ b/Misc/python-config.sh.in
@@ -46,7 +46,7 @@ LIBM="@LIBM@"
 LIBC="@LIBC@"
 SYSLIBS="$LIBM $LIBC"
 ABIFLAGS="@ABIFLAGS@"
-LIBS="@LIBPYTHON@ @LIBS@ $SYSLIBS"
+LIBS="@MODULE_LDFLAGS@ @LIBS@ $SYSLIBS"
 LIBS_EMBED="-lpython${VERSION}${ABIFLAGS} @LIBS@ $SYSLIBS"
 BASECFLAGS="@BASECFLAGS@"
 LDLIBRARY="@LDLIBRARY@"
diff --git a/Misc/python.pc.in b/Misc/python.pc.in
index 027dba3858..c2c740e82b 100644
--- a/Misc/python.pc.in
+++ b/Misc/python.pc.in
@@ -9,5 +9,5 @@ Description: Build a C extension for Python
 Requires:
 Version: @VERSION@
 Libs.private: @LIBS@
-Libs: -L${libdir} @LIBPYTHON@
+Libs: -L${libdir} @MODULE_LDFLAGS@
 Cflags: -I${includedir}/python@VERSION@@ABIFLAGS@
